tests/components/jellyfin/test_config_flow.py

```diff
@@ -113,7 +109,7 @@ async def test_form_invalid_auth(
     assert result["type"] == "form"
     assert result["errors"] == {}
 
-    mock_client.auth.login.return_value = MOCK_UNSUCCESFUL_LOGIN_RESPONSE
+    mock_client.auth.login.return_value = load_json_fixture("auth-login-failure.json")
```

Comment:
You could make an `async_load_json_fixture` that does it in the executor

Reply:
I think I did that properly with the bind _hass?